nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -490,6 +502,8 @@ in
         restartTriggers = [ nixConf ];
       };
 
+    systemd.tmpfiles.rules = [ "d ${cfg.buildLocation} 0775 root root -" ];
```

Comment:
This breaks write permissions for non-root on `/tmp` with the default value and `boot.tmpOnTmpfs = true;`
```
setting up tmpfiles
[/etc/tmpfiles.d/tmp.conf:11] Duplicate line for path "/tmp", ignoring.
```
```
$ getfacl /tmp
getfacl: Removing leading '/' from absolute path names
# file: tmp/
# owner: root
# group: root
user::rwx
group::rwx
other::r-x
```